functions/src/modules/Users.functions.ts

```diff
@@ -1,62 +1,162 @@
 import * as functions from 'firebase-functions';
 import * as admin from 'firebase-admin';
+import GoogleSheet from '../services/GoogleSheets';
+import { withDefault } from '../utils/parsers';
 
 const db = admin.database();
 
+enum RegistrationColumns {
+  Details = 'Details',
+  Status = 'Status',
+  Timestamp = 'Timestamp',
+  Name = 'Name',
+  Country = 'Country',
+  EmailAddress = 'Email Address',
+  PhoneNumber = 'Phone Number',
+  WhatsApp = 'WhatsApp',
+  Languages = 'Languages',
+  Services = 'Services',
+  Experience = 'Experience',
+  Influencer = 'Influencer',
+  RecommendedBy = 'Recommended By',
+}
+
+enum Roles {
+  CR = 'CR',
+  FC = 'FC',
+  QC = 'QC',
+  TE = 'TE',
+  SE = 'SE',
+  SQR = 'SQR',
+}
+
+enum Decision {
+  Yes = 'Yes',
+  No = 'No',
+}
+
+/**
+ * 1. Scan the Registrations Google Sheet and exports any new users to database
+ * 2. Do not import if user email already exists
+ *
+ */
+export const importUserRegistrationData = functions.https.onRequest(
+  async (req: functions.Request, res: functions.Response) => {
+    const gsheets: GoogleSheet = new GoogleSheet(
+      functions.config().user.spreadsheet_id,
```

Comment:
```suggestion
      functions.config().registrations.spreadsheet_id,
```
And please document this in `README`.

Reply:
Done.